docs/nextConfigDocsInfra.js

```diff
@@ -5,16 +5,35 @@ if (process.env.CONTEXT === 'deploy-preview') {
   DEPLOY_ENV = 'pull-request';
 }
 
+/**
+ * ===================================================================================
+ * Fun facts about the `process.env.CONTEXT`:
+ *
+ * - It is defined by Netlify environment variables (https://docs.netlify.com/configure-builds/environment-variables/#build-metadata)
+ * - `process.env.CONTEXT === 'production'` does not mean mui.com build, it is the site configuration.
+ *   For example, the `master` branch of the Core team is considered a `production` build on Netlify based on https://app.netlify.com/sites/material-ui/settings/deploys#branches.
+ *
+ * - Each team has different site https://app.netlify.com/teams/mui/sites. Make sure to checkout the right site before making any changes.
+ *
+ */
 if (process.env.CONTEXT === 'production' || process.env.CONTEXT === 'branch-deploy') {
   DEPLOY_ENV = 'production';
 }
 
+if (process.env.CONTEXT === 'production' && process.env.HEAD === 'master') {
+  // For the Core team site, visit https://app.netlify.com/sites/material-ui/settings/deploys#branches for the config details.
+  DEPLOY_ENV = 'staging';
+}
```

Comment:
@oliviertassinari I have to make the change to this, otherwise, https://material-ui.netlify.app/experiments/ does not work.

For the core repo, https://app.netlify.com/sites/material-ui/settings/deploys#branches, `master` branch has `process.env.CONTEXT === 'production'` (not `branch-deploy`).